### DISCUSSION TITLE
[Beta] For glibc-2.28 or newer, x86_64 architecture.

### DISCUSSION THREAD

**Discussion by ytyou** [ytyou/ticktock#43] [General]
Contains various bug fixes.

<hr /><em>This discussion was created from the release <a href='https://github.com/ytyou/ticktock/releases/tag/v0.11.6'>[Beta] For glibc-2.28 or newer, x86_64 architecture.</a>.</em>

**Comment by ylin30** [comment]
Let's also provide ARM 32 and 64 bit binaries.
